extensions/HyperVExtension/src/HyperVExtension/Strings/en-US/Resources.resw

```diff
@@ -201,6 +201,18 @@
     <value>Unable to start the operation because it is already in progress</value>
     <comment>Error text to tell the user that the process to create the virtual machine is already in progress.</comment>
   </data>
+  <data name="FailedToReadGuestOsProperties" xml:space="preserve">
+    <value>Failed to read guest OS properties</value>
+    <comment>Error text to tell the user that an error happened while trying to read virtual machine's OS properties.</comment>
+  </data>
+  <data name="FailedToDetermineGuestOsVersion" xml:space="preserve">
+    <value>Failed to determine guest OS version</value>
+    <comment>Error text to tell the user that an error happened while trying to determine virtual machine's OS version.</comment>
+  </data>
+  <data name="GuestOsOperationNotSupported" xml:space="preserve">
+    <value>This operation is not supported for the guest OS</value>

```

Comment:
Should we say which version we support in the error text as well? And the one above it

Reply:
I planned it to be a generic message for other situations, but yes, I can add something like "The operation requires {0}."